web/packages/teleport/src/Bots/List/ActionCell.tsx

```diff
@@ -0,0 +1,33 @@
+/**
+ * Teleport
+ * Copyright (C) 2024 Gravitational, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React from 'react';
+import { Cell } from 'design/DataTable';
+import { MenuButton, MenuItem } from 'shared/components/MenuAction';
+
+import { BotOptionsCellProps } from 'teleport/Bots/types';
+
+export function BotOptionsCell({ onClickDelete }: BotOptionsCellProps) {
+  return (
+    <Cell align="right">
+      <MenuButton>
+        <MenuItem onClick={onClickDelete}>Delete...</MenuItem>
```

Comment:
As per @roraback the `...` indicates that a menu item **does not** take an immediate action, rather it will open a follow up window/option (in this case a confirmation modal)